app/Http/Controllers/ContactController.php

```diff
@@ -2,12 +2,47 @@
 
 namespace App\Http\Controllers;
 
+use App\Http\Request\ContactRequest;
 use Illuminate\Http\Request;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Mail\Message;
 
 class ContactController extends Controller
 {
     public function index(){
         return view('home.contact');
     }
+
+    public function send(ContactRequest $request):RedirectResponse
+    {
+        $callback = function (string $test): string{
+            return "{$test} input";
+        };
+
+
+        $data = $request->validated();
+        \Log::debug( 'test', $data);
+
+        \Mail::send('emails.contact',
+            [
+                'firstname' => $data['firstname'],
+                'lastname' => $data['lastname'],
+                'email' => $data['email'],
+                'phonenumber' => $data['phonenumber'],
+                'subject' => $data['subject'],
+                'messageText' => $data['message'],
+            ],
+        
+        function(Message $message) use ($data){ 
+            $message->subject('mesage from ' . $data['email']);
+            $message->to('tech@baloon.app');
```

Comment:
Please adjust `to` and `from` methods to use your project related values.

Reply:
changed